hubresolver/README.md

```diff
@@ -0,0 +1,98 @@
+# Hub Resolver
+
+Use resolver type `hub`.
+
+## Parameters
+
+| Param Name       | Description                                                                   | Example Value                                              |
+|------------------|-------------------------------------------------------------------------------|------------------------------------------------------------|
+| `kind`           | Either `task` or `pipeline`                                                   | `task`                                                     |
+| `name`           | The name of the task or pipeline to fetch from the hub                        | `golang-build`                                             |
+| `version`        | Version of task or pipeline to pull in from hub. Wrap the number in quotes!   | `"0.5"`                                                    |
```

Comment:
we should also add a `catalog` field because the hub supports different catalogs

Reply:
Agreed! We can default that to `Tekton`

Reply:
Good point, let's add it